model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -29,6 +30,10 @@ import java.io.File
  * The configuration model for all ORT components.
  */
 data class OrtConfiguration(
+    /**
+     * The configuration of the scanner.
+     */
+    val analyzer: AnalyzerConfiguration? = null,
```

Comment:
I like that you make the analyzer configurable in the config file, but this should happen in a separate commit.

Reply:
The `reference.conf` needs to be updated to include the analyzer configuration. For reference, and to have at least a basic test for it in `OrtConfigurationTest`. Also, the values should be used in the analyzer. To correctly implement that I think the `--allow-dynamic-versions` and `--ignore-tool-versions` command line parameters of the analyzer should be removed, because they can now also be set using `-P`. @sschuberth Any opinion on that?

Reply:
Missing blank line after this line.